design/cli-install-changes.md

```diff
@@ -0,0 +1,286 @@
+      --client-burst int                           maximum number of requests by the server to the Kubernetes API in a short period of time (default 30)
+      --client-qps float32                         maximum number of requests per second by the server to the Kubernetes API once the burst limit has been reached (default 20)
+      --default-backup-ttl duration                how long to wait by default before backups can be garbage collected (default 720h0m0s)
+      --disable-controllers strings                list of controllers to disable on startup. Valid values are backup,backup-sync,schedule,gc,backup-deletion,restore,download-request,restic-repo,server-status-request
+  -h, --help                                       help for server
+      --log-format                                 the format for log output. Valid values are text, json. (default text)
+      --log-level                                  the level at which to log. Valid values are debug, info, warning, error, fatal, panic. (default info)
+      --metrics-address string                     the address to expose prometheus metrics (default ":8085")
+      --plugin-dir string                          directory containing Velero plugins (default "/plugins")
+      --profiler-address string                    the address to expose the pprof profiler (default "localhost:6060")
+      --restore-only                               run in a mode where only restores are allowed; backups, schedules, and garbage-collection are all disabled. DEPRECATED: this flag will be removed in v2.0. Use read-only backup storage locations instead.
+      --restore-resource-priorities strings        desired order of resource restores; any resource not in the list will be restored alphabetically after the prioritized resources (default [namespaces,storageclasses,persistentvolumes,persistentvolumeclaims,secrets,configmaps,serviceaccounts,limitranges,pods,replicaset,customresourcedefinitions])
+      --terminating-resource-timeout duration      how long to wait on persistent volumes and namespaces to terminate during a restore before timing out (default 10m0s)
+
+```
+
+2) `velero backup-location`
+Commands/flags for backup locations. 
+
+```
+     set 
+        --default string                                  sets the default backup storage location (default "default") (NEW, -- was `server --default-backup-storage-location) 
+
+      create
+        --dry-run                                         generate resources, but don't send them to the cluster. Use with -o. Optional. (NEW)
+        --default                                         Sets this new location to be the new default backup location. Default is false. (NEW) 
+        --secret-file string                              file containing credentials for backup provider. If not specified, set --no-secret must be used for confirmation. Optional. (MOVED FROM install)
```

Comment:
I think `--secret-file` and related flags probably needs to move to the `velero plugin add` command, since we currently only support a single secret per provider.

Reply:
Argh, I spaced out on this. Yep. Glad you noticed it.